storage/src/main/java/com/tencent/rss/storage/handler/impl/ComposedClientReadHandler.java

```diff
@@ -47,6 +56,25 @@ public ComposedClientReadHandler(ClientReadHandler... handlers) {
     if (size > 2) {
       this.coldDataReadHandler = handlers[2];
     }
```

Comment:
Do we need a check of instanceof ?
It because the handler's order may be wrong.

Reply:
No need.StorageType is speciffic.